controller/tap/apiserver.go

```diff
@@ -23,7 +22,13 @@ type apiServer struct {
 }
 
 // NewAPIServer creates a new server that implements the Tap APIService.
-func NewAPIServer(addr string, cert tls.Certificate, k8sAPI *k8s.API, client pb.TapClient, disableCommonNames bool) (*http.Server, net.Listener, error) {
+func NewAPIServer(
+	addr string,
+	cert tls.Certificate,
+	k8sAPI *k8s.API,
+	grpcTapServer *GRPCTapServer,
```

Comment:
can this depend on an interface? or does it need to depend on the concrete GRPCTapServer type?

Reply:
I changed this to expect a `pb.TabServer` instead of a concrete type.